workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -243,6 +248,16 @@ public void SetComponentWriteAccess(uint componentId, string attribute)
                 writePermissions[componentId] = attribute;
             }
 
+            public string GetComponentWriteAccess(uint componentId)
```

Comment:
There is nothing from the name of these functions that tells me what the difference is.

Reply:
I based them to be similar to the other methods near them

Reply:
See #629 as this is changed. Do we need a `Get` for this?

Reply:
The get was necessary to prevent unneeded modification of this property

Reply:
A user would not want to check / keep track of "did I write to this thing before?"

Reply:
So the logic was: if the write access is set, don't modify it, else set it?

Reply:
yes